From edb292317b3c2152f306aae80a0aae27e8db3a37 Mon Sep 17 00:00:00 2001
From: zhangmeng <zhangmeng.kevin@spacemit.com>
Date: Sat, 16 Mar 2024 18:57:11 +0800
Subject: [PATCH 0502/1451] update evb dts:

1. remove clk_ignore_unused from bootargs
2. modify rdinit=/init.tmp to rdinit=/init

Change-Id: Ic8a95741a1bd4195a7a6233c046f871499585e48
---
 arch/riscv/boot/dts/spacemit/k1-x_evb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/spacemit/k1-x_evb.dts b/arch/riscv/boot/dts/spacemit/k1-x_evb.dts
index f34564514ce1..99156c1188d8 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_evb.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_evb.dts
@@ -112,7 +112,7 @@ dpu_resv: dpu_reserved@27000000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 clk_ignore_unused rdinit=/init.tmp";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
-- 
2.47.0

